.stylish-haskell.yaml

```diff
@@ -144,15 +167,23 @@ steps:
 # to. Different steps take this into account. Default: 80.
 columns: 80
 
+# By default, line endings are converted according to the OS. You can override
+# preferred format here.
+#
+# - native: Native newline format. CRLF on Windows, LF on other OSes.
+#
+# - lf: Convert to LF ("\n").
+#
+# - crlf: Convert to CRLF ("\r\n").
+#
+# Default: native.
+newline: native
+
 # Sometimes, language extensions are specified in a cabal file or from the
 # command line instead of using language pragmas in the file. stylish-haskell
 # needs to be aware of these, so it can parse the file correctly.
 #
 # No language extensions are enabled by default.
-language_extensions:
-  - RecordWildCards
-  - OverloadedStrings
-  - LambdaCase
-  - MultiParamTypeClasses
-  - BangPatterns
-  - FlexibleContexts
+# language_extensions:
+  # - TemplateHaskell
+  # - QuasiQuotes
```

Comment:
Why remove them?

Reply:
I re-dumped the defaults since the old defaults were for an old version of SH and wer ebreaking it for some reason.